scipy/special/_special_ufuncs_docs.cpp

```diff
@@ -4762,6 +4762,42 @@ const char *mathieu_cem_doc = R"(
     --------
     mathieu_a, mathieu_b, mathieu_sem
 
+    Notes
+    -----
+    Even Mathieu functions are the solutions to Mathieu's differential equation
+
+    .. math::
+
+        \frac{d^2y}{dx^2} + (a_m - 2q \cos(2x))y = 0
+
+    for which the characteristic number :math:`a_m` results in an even, periodic
+    solution :math:`y(x)`` with period 180 degrees (for even :math:`m`) or
+    360 degrees (for odd :math:`m`).
+
+    References
+    ----------
+    'Mathieu function'. *Wikipedia*.
```

Comment:
Is this valid reference format? I expect:

```
.. [1] O. McNoleg, ...
```

But perhaps SciPy follows its own standard.

Reply:
Is McNoleg the author of this article?

Reply:
Oh! I messed up. I'll fix those.